tests/test_wfnet.py

```diff
@@ -69,7 +69,6 @@ def wf(net_factory, mol):
         mole = pyscf.gto.M(atom=mol.as_pyscf(), unit='bohr', basis='6-311g', cart=True)
         basis = GTOBasis.from_pyscf(mole)
         args += (basis,)
-        kwargs.update({'jastrow_factory': JastrowNet, 'dist_feat_dim': 4})
```

Comment:
I think with this removal, the test tests only a non-trainable ansatz, isn't that right? This should be rather replaced by some stripped-down version of the omni.